pkg/kubelet/kubelet_server_journal_linux.go

```diff
@@ -0,0 +1,40 @@
+// +build linux
```

Comment:
CI verify didn't like it when I used `!windows` here, which is why I had to create the darwin stub-implementation of the function